app/templates/admin/registered_users.html

```diff
@@ -45,24 +42,37 @@ <h2 class="ui header">
                             <th class="sorted ascending">Last name</th>
                             <th>Email address</th>
                             <th>Account type</th>
+                            <th>Donor Level</th>
                         </tr>
                     </thead>
                     <tbody>
-                    {% for u in users | sort(attribute='last_name') %}
-                        <tr onclick="window.location.href = '{{ url_for('admin.user_info', user_id=u.id) }}';">
-
-                            <td>{{ u.first_name }}</td>
-                            <td>{{ u.last_name }}</td>
-                            <td>{{ u.email }}</td>
-                            <td class="user role">{{ u.role.name }}</td>
-                        </tr>
-                    {% endfor %}
+                        {% for u in users | sort(attribute='last_name') %}
+                            <tr>
+                                <td onclick="window.location.href = '{{ url_for('admin.user_info', user_id=u.id) }}';">{{ u.first_name }}</td>
+                                <td onclick="window.location.href = '{{ url_for('admin.user_info', user_id=u.id) }}';">{{ u.last_name }}</td>
+                                <td onclick="window.location.href = '{{ url_for('admin.user_info', user_id=u.id) }}';">{{ u.email }}</td>
```

Comment:
check out the way I did the `<a>` tag above and do the same for the other `<td>` tags